engine/classes/Elgg/UpgradeService.php

```diff
@@ -144,9 +176,6 @@ protected function upgradeCode($version, $quiet = false) {
 	 * @return mixed
 	 */
 	protected static function includeCode($file) {
-		// do not remove - some upgrade scripts depend on this
-		global $CONFIG;
```

Comment:
I assumed it was needed just for the 2.0 upgrade but not the 2.1 upgrade. In any case, I'm planning a completely new way to run upgrades, so at least then this won't be needed anymore.

I can add it back if it's still needed for the time being.